server/routes/visitJourney/selectVisitors.ts

```diff
@@ -96,6 +97,17 @@ export default class SelectVisitors {
       restriction => restriction.restrictionType === 'CLOSED',
     )
 
+    selectedVisitors.forEach(visitor => {
```

Comment:
Could probably simplyfy this by passing the restrictions for **all visitors** in one go to `getBanStatus()`. E.g. (untested!):
```
const allSelectedVisitorBans: Restriction[] = selectedVisitors.reduce((_acc, visitor) => {
  return visitor.restrictions
}, [])

const banStatus = getBanStatus(allSelectedVisitorBans)
visitSessionData.daysUntilBanExpiry = banStatus.isBanned ? banStatus.numDays : undefined
```